draft-ietf-quic-recovery.md

```diff
@@ -598,8 +598,8 @@ discarded.
 
 When a PTO timer expires, a sender MUST send at least one ack-eliciting packet
 as a probe, unless there is no data available to send.  An endpoint MAY send up
-to two ack-eliciting packets, to avoid an expensive consecutive PTO expiration
-due to a single packet loss.
+to two full sized datagrams containing ack-eliciting packets, to avoid an
+expensive consecutive PTO expiration due to a single lost datagram.
 
```

Comment:
Where is that limit of two specified? Here? Why?

Reply:
If you send 1, it might be lost.  Ideally, fewer is better, so the question of 1 or 2 was left to implementers.

Reply:
I mean, there is no limit to coalescing AFAIK, so the limit of two here seems arbitrary, albeit probably very sensible.
https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.12.2

Reply:
There's no limit of 2 packets in a datagram here and I don't think there's one elsewhere?

Reply:
> An endpoint MAY send up to two full-sized datagrams containing ack-eliciting packets,

I read that as it MAY NOT send more than two, but it MAY send less.

Reply:
Ah, I am confusing UDP datagrams and packets.